homeassistant/components/climate/__init__.py

```diff
@@ -379,8 +379,8 @@ def state_attributes(self):
         data = {
             ATTR_CURRENT_TEMPERATURE:
             self._convert_for_display(self.current_temperature),
-            ATTR_MIN_TEMP: self._convert_for_display(self.min_temp),
```

Comment:
This is wrong. These values will now no longer be converted when necessary. Please remove this change.